portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,8 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+      ['I love music!', 'I have a fake tooth :(',
+       'Soccer is my favorite sport.', 'I once won a "Most talkative" award 😞'];
```

Comment:
Haha, most talkative isn't a bad thing! I love seeing you engage in team meetings!